odoo-shippable/scripts/build-image.sh

```diff
@@ -121,14 +124,22 @@ git_clone_execute "${OH_MY_ZSH_REPO}" "master" "tools/install.sh"
 git_clone_copy "${ZSH_THEME_REPO}" "master" "schminitz.zsh-theme" "~/.oh-my-zsh/themes/odoo-shippable.zsh-theme"
 sed -i 's/robbyrussell/odoo-shippable/g' ~/.zshrc
 
-# Install & configure vim
+# Upgrade & configure vim
+apt-get upgrade vim
+wget -q -O /usr/share/vim/vim80/spell/es.utf-8.spl http://ftp.vim.org/pub/vim/runtime/spell/es.utf-8.spl
 git_clone_execute "${SPF13_REPO}" "3.0" "bootstrap.sh"
 git_clone_copy "${VIM_OPENERP_REPO}" "master" "vim/" "~/.vim/bundle/vim-openerp"
-wget -q -O /usr/share/vim/vim74/spell/es.utf-8.spl http://ftp.vim.org/pub/vim/runtime/spell/es.utf-8.spl
```

Comment:
Yes! But `git_clone_execute "${SPF13_REPO}" "3.0" "bootstrap.sh"` opens vim and if not fixed `es.utf-8.spl` caused this error `/usr/share/vim/vim80/spell/es.utf-8.spl: No such file or directory`